brian2cuda/tests/test_gpu_detection.py

```diff
@@ -53,15 +53,15 @@ def test_manual_setting_compute_capability():
 @attr('cuda_standalone', 'standalone-only')
 @with_setup(teardown=reinit_devices)
 def test_unsupported_compute_capability_error():
-    prefs.codegen.generators.cuda.compute_capability = 2.0
+    prefs.devices.cuda_standalone.brian_backend.compute_capability = 2.0
     with assert_raises(NotImplementedError):
         run(0*ms)
 
 
 @attr('cuda_standalone', 'standalone-only')
 @with_setup(teardown=reinit_devices)
 def test_warning_compute_capability_set_twice():
-    prefs.codegen.generators.cuda.compute_capability = 3.5
+    prefs.devices.cuda_standalone.brian_backend.compute_capability = 3.5
     prefs.codegen.cuda.extra_compile_args_nvcc.append('-arch=sm_37')
```

Comment:
Also this one is missing. I think they are defined in `cuda_generator.py`. Should also go into `cuda_prefs.py` and be named `prefs.devices.cuda_standalone.cuda_backend.extra_compile_args_nvcc`.

Reply:
This was already in `cuda_prefs.py` in the `preference` bucket (`codegen.cuda`) , hence kept it like that ? Should I transfer it to the new bucket `cuda_backend`.

Reply:
Yes, that is good, no need for `codegen.cuda` anymore, everything should be under `devices.cuda_standalone`.